src/external/wannier_shared.jl

```diff
@@ -209,31 +209,31 @@ end
 
 @doc raw"""
 Computes the matrix ``[M^{k,b}]_{m,n} = \langle u_{m,k} | u_{n,k+b} \rangle``
-for given `k`, `kpb` = ``k+b``.
+for given `k`.
 
 `G_shift` is the "shifting" vector, correction due to the periodicity conditions
 imposed on ``k \to  ψ_k``.
-It is non zero if `kpb` is taken in another unit cell of the reciprocal lattice.
+It is non zero if `k_plus_b` is taken in another unit cell of the reciprocal lattice.
 We use here that:
 ``u_{n(k + G_{\rm shift})}(r) = e^{-i*\langle G_{\rm shift},r \rangle} u_{nk}``.
 """
-@views function overlap_Mmn_k_kpb(basis::PlaneWaveBasis, ψ, ik, ikpb, G_shift, n_bands)
-    # Search for common Fourier modes and their resp. indices in Bloch states k and kpb
+@views function overlap_Mmn_k_kpb(basis::PlaneWaveBasis, ψ, ik, ik_plus_b, G_shift, n_bands)
+    # Search for common Fourier modes and their resp. indices in Bloch states k and k+b
     # TODO Check if this can be improved using the G vector mapping in the kpoints
-    k   = basis.kpoints[ik]
-    kpb = basis.kpoints[ikpb]
-    equivalent_G_vectors = [(iGk, index_G_vectors(basis, kpb, Gk + G_shift))
-                            for (iGk, Gk) in enumerate(G_vectors(basis, k))]
-    iGk   = [eqG[1] for eqG in equivalent_G_vectors if !isnothing(eqG[2])]
-    iGkpb = [eqG[2] for eqG in equivalent_G_vectors if !isnothing(eqG[2])]
+    k        = basis.kpoints[ik]
+    k_plus_b = basis.kpoints[ik_plus_b]
+    equivalent_G_vectors = [(ip, index_G_vectors(basis, k_plus_b, p + G_shift))
+                            for (ip, p) in enumerate(G_vectors(basis, k))]
+    ip        = [eqG[1] for eqG in equivalent_G_vectors if !isnothing(eqG[1])]
```

Comment:
I don't think that is a mistake, because the first entry in a tuple is just an index that is never nothing, but `index_G_vectors` can be nothing (if that `G` vector is not used in the `k`-point specific basis.